### PR TITLE
fix(components): add subcomponent property syntax to Tooltip configuration

### PR DESCRIPTION
**Pull request by geotrev**
Forgot this in #612 😶 

Screenshot 2024-08-06 at 3 56 23 PM

Screenshot 2024-08-06 at 4 07 59 PM

### CODE REVIEW

**Review by ze-flo — APPROVED** [review]
Good catch!

File: src/pages/components/tooltip.mdx [hunk 1]
```diff
@@ -86,24 +86,24 @@ import TypeCode from '!!raw-loader!../../examples/tooltip/TooltipType.tsx';
...
 ## API
 
-### Paragraph
+### Tooltip
+
+<Component components={props.data.mdx.components} componentName="tooltip" />
+
+<PropSheet components={props.data.mdx.components} componentName="tooltip" />
+
+### Tooltip.Paragraph
```
Comment (jzempel) [inline src/pages/components/tooltip.mdx:95]:
```suggestion
#### Tooltip.Paragraph
```

Subcomponents are H4

File: src/pages/components/tooltip.mdx [hunk 2]
```diff
@@ -86,24 +86,24 @@ import TypeCode from '!!raw-loader!../../examples/tooltip/TooltipType.tsx';
...
 
 <Component components={props.data.mdx.components} componentName="paragraph" />
 
 <PropSheet components={props.data.mdx.components} componentName="paragraph" />
 
-### Title
+### Tooltip.Title
```
Comment (jzempel) [inline src/pages/components/tooltip.mdx:101]:
```suggestion
#### Tooltip.Title
```

**Review by jzempel — CHANGES_REQUESTED** [review]

**Review by jzempel — APPROVED** [review]

**Comment by geotrev** [comment]
Waiting for `displayName` changes from `next.21` so the subcomponent values are correct.

**Comment by ze-flo** [comment]
> Waiting for `displayName` changes from `next.21` so the subcomponent values are correct.

Handled in https://github.com/zendeskgarden/website/pull/621

**Comment by ze-flo** [comment]
This PR might be redundant once https://github.com/zendeskgarden/website/pull/621 merges.

**Comment by geotrev** [comment]
Closing given the updates pushed in #621

**Pull request closed without merging**